### PR TITLE
Adds the last elements to styles dev

### PR DESCRIPTION
**Pull request by sarahghp**
Not all of these are styled correctly, but I think it's a good start. Fringe PR to follow.

### CODE REVIEW

File: dev/oranda.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "homepage": "https://oranda.axo.dev",
   "readme_path": "dev/README.md",
+  "theme": "light",
```
Comment (SaraVieira) [inline dev/oranda.json:4]: Would you mind defaulting to black?

Comment (sarahghp) [reply dev/oranda.json:4]: So the reason I did this is to make sure when people are testing dev stuff, they don't forget to test the theme option and they don't forget to consider light mode. I would prefer not to change it, but I am open to reasons I haven't considered.

File: dev/README.md [hunk 2]
```diff
@@ -50,6 +52,9 @@ One error we ran into before was in code-blocks with HTML specifically, so let's
 ![An image from a vintage video synth](./static/some_art.png)
 Here is an image I made.
 
+Here is one with an external link
+![](https://sarahghp.com/vids/noodling/noodl06)
```
Comment (sarahghp) [inline dev/README.md]: Also, I almost forgot, we should link to something that is not mine. Do you have a suggestion for a stable remote link?

Comment (SaraVieira) [reply dev/README.md]: We can use unsplash images, they don't get deleted

https://unsplash.com/photos/J4CeJunkE8M/download?ixid=MnwxMjA3fDB8MXxhbGx8MTR8fHx8fHwyfHwxNjc0NTYzNDA0&force=true&w=640

File: dev/oranda.json (hunk 1 above)
Comment (SaraVieira) [reply dev/oranda.json:4]: oh this is the dev yeah sorry

File: dev/README.md (hunk 2 above)
Comment (sarahghp) [reply dev/README.md]: Updated with pexels, since it is rights free. Thank you for the suggestion!

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,3 @@
 /target
 public/
-
+**/.DS_Store
```
Comment (ashleygwilliams) [inline .gitignore:3]: this is an OS specific file and should be in a global gitignore, not a project gitignore

File: dev/additional.md [hunk 4]
```diff
@@ -1,66 +1,13 @@
...
-brown and white in trees,
-snows and skies of laughter
-always diminishing, less funny
-not just darker, not just grey.
-
-It may be the coldest day of
-the year, what does he think of
-that? I mean, what do I? And if I do,
-perhaps I am myself again.
+<h2 style="margin-bottom:0">Snail</h2>
```
Comment (ashleygwilliams) [inline dev/additional.md:1]: is this inline style on purpose?

**Review by SaraVieira — APPROVED** [review]

**Review by ashleygwilliams — CHANGES_REQUESTED** [review]

**Pull request merged**